### ISSUE TITLE
Staging Review finding: Email notifications have sections announced by screen readers as data tables

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94351]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Forms Status
**Feature:** Form Status on My VA & Confirmation Page

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In each of the email notifications, the gray box containing submission details may be announced as a data table by some screen readers. This can be confusing for screen reader users, who may expect tabular data when they hear the table semantics but will instead just hear lots of text in one cell.

## Recommended action

When using tables for layouts in HTML emails, make sure they have `role="presentation"`. The outermost tables wrapping the message content do this, but any other tables nested within that also need `role="presentation"`.

I believe Jeana has already [logged this with the VA Notify team](https://github.com/department-of-veterans-affairs/notification-utils/issues/178).

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
This has been fixed

<img width="695" alt="Image" src="https://github.com/user-attachments/assets/b045db89-f032-4c11-9119-8d86c7adbdf4" />
